src/frontend/screens/Settings/CreateOrJoinProject/CreateProject/index.tsx

```diff
@@ -110,37 +125,80 @@ export const CreateProject: NativeNavigationComponent<'CreateProject'> = ({
     defaultValues: {projectName: ''},
   });
 
-  function handleCreateProject(val: ProjectFormType) {
+  const handleCreateProject = async (val: ProjectFormType) => {
```

Comment:
this should no longer be async.

Also can we rename it to `handleCreateOrUpdateProject`